lstms/lstms_env.yml

```diff
@@ -0,0 +1,221 @@
+name: capstone
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - appnope=0.1.0=py37_0
+  - argon2-cffi=20.1.0=py37haf1e3a3_1
+  - attrs=20.2.0=py_0
+  - backcall=0.2.0=py_0
+  - basemap=1.2.2=py37h2339127_0
+  - blas=1.0=mkl
+  - bleach=3.2.1=py_0
+  - boost-cpp=1.72.0=hdf9ef73_0
+  - branca=0.4.1=py_0
+  - brotlipy=0.7.0=py37h60d8a13_1000
+  - bzip2=1.0.8=haf1e3a3_3
+  - c-ares=1.16.1=haf1e3a3_3
+  - ca-certificates=2020.6.20=hecda079_0
+  - cairo=1.16.0=hec6a9b0_1003
+  - certifi=2020.6.20=py37hc8dfbb8_0
+  - cffi=1.14.2=py37hed5b41f_0
+  - cfitsio=3.470=hdf94aef_6
+  - chardet=3.0.4=py37hc8dfbb8_1007
+  - click=7.1.2=pyh9f0ad1d_0
+  - click-plugins=1.1.1=py_0
+  - cligj=0.5.0=py_0
+  - cryptography=3.1=py37h94e4008_0
+  - curl=7.71.1=hcb81553_6
+  - cycler=0.10.0=py37_0
+  - dbus=1.13.16=h18a8e69_0
+  - decorator=4.4.2=py_0
+  - defusedxml=0.6.0=py_0
+  - descartes=1.1.0=py_4
+  - entrypoints=0.3=py37_0
+  - expat=2.2.9=hb1e8313_2
+  - ffmpeg=4.3.1=h9903f1f_0
+  - fiona=1.8.17=py37h8cd0b58_0
+  - folium=0.11.0=py_0
+  - fontconfig=2.13.1=h6b1039f_1001
+  - freetype=2.10.2=ha233b18_0
+  - freexl=1.0.5=h0b31af3_1002
+  - gdal=3.1.2=py37h85de8e7_2
+  - geopandas=0.8.1=py_0
```

Comment:
There's a lot of geospatial stuff in this env dump that's not needed, but should be enough to get started coding with